sdks/python/apache_beam/pipeline.py

```diff
@@ -694,6 +694,10 @@ def apply(
       if auto_unique_labels:
         # If auto_unique_labels is set, we will append a unique suffix to the
         # label to make it unique.
+        logging.warning(
+            'Using --auto_unique_labels could cause data loss when '
```

Comment:
Should we put a note in the docstring of this flag as well?

Reply:
Good idea.